src/edit.js

```diff
@@ -201,20 +210,62 @@ function Edit( props ) {
 		return <div { ...blockProps }>{ mediaPlaceholder }</div>;
 	}
 
+	const skinPreviewUrl = ( skin ) => {
+		if ( skin ) {
+			const match = skin.match(
+				/(?:https?:)?(?:\/\/)?skins\.webamp\.org\/skin\/(\w+)\/(?:.*)?/
+			);
+	
+			if ( match && match.length === 2 ) {
+				return `https://cdn.webampskins.org/screenshots/${ match[ 1 ] }.png`;
+			}
+		}
+	}
+
 	return (
 		<>
 			<InspectorControls>
 				<PanelBody title={ __( 'Winamp Player Skin', 'winamp-block' ) }>
-					<TextControl
-						label={ __( 'Skin URL', 'winamp-block' ) }
-						help={ __(
-							'The URL of the player skin to use. Find skins at https://skins.webamp.org/.',
-							'winamp-block'
-						) }
-						value={ currentSkin }
-						onChange={ ( skin ) =>
-							setAttributes( { currentSkin: skin } )
-						}
+					{ useCustomUrl ? (
+						<TextControl
+							label={ __( 'Skin URL', 'winamp-block' ) }
+							help={ __(
+								'The URL of the player skin to use. Find skins at https://skins.webamp.org/.',
+								'winamp-block'
+							) }
+							value={ currentSkin }
+							onChange={ ( skin ) =>
+								setAttributes( { currentSkin: skin } )
+							}
+						/>
+						)
+						:
+						(
+						<div className="preview-wrapper">
+							{ defaultSkins.length && defaultSkins.map( ( skin, index ) => {
+								return (
+									<label key={ index } className="winamp-radio-wrapper">
```

Comment:
Would be good to add `htmlFor` to associate with the following input element.

Reply:
@Sidsector9 Done as well.